adapters/tappx/tappx_test.go

```diff
@@ -2,10 +2,31 @@ package tappx
 
 import (
 	"github.com/prebid/prebid-server/adapters/adapterstest"
+	"github.com/prebid/prebid-server/openrtb_ext"
+	"github.com/stretchr/testify/assert"
 	"net/http"
+	"regexp"
 	"testing"
 )
 
 func TestJsonSamples(t *testing.T) {
 	adapterstest.RunJSONBidderTest(t, "tappxtest", NewTappxBidder(new(http.Client), "https://{{.Host}}"))
 }
+
+func TestTsValue(t *testing.T) {
+	adapter := NewTappxBidder(new(http.Client), "https://{{.Host}}")
+	var test int
+	test = 0
+	var tappxExt openrtb_ext.ExtImpTappx
+	tappxExt.Host = "example.host.tappx.com"
+	tappxExt.Endpoint = "DUMMYENDPOINT"
+	tappxExt.TappxKey = "dummy-tappx-key"
+
+	url, err := adapter.buildEndpointURL(&tappxExt, test)
+
+	match, err := regexp.MatchString(`&ts=[0-9]{13}`, url)
```

Comment:
I'd say assert the url to the actual expected string rather than doing a regexp match

Reply:
The regex match was purposely added in order to validate the ts parameter. You can read more about this in one of the first requested changes in this pull request by @guscarreon .
The full URL cannot be asserted since ts value is dynamic. I will add the rest of the URL to the regexp match to make it more robust but I think this cannot be  skipped.

Reply:
Ah! I missed the `ts` part before. Makes sense. Thanks for the clarification :)